api/swagger.yaml

```diff
@@ -1610,6 +1610,13 @@ definitions:
       ID:
         type: "string"
       Version:
+        description: |
+          The version number of the node object. This is required to avoid conflicting writes.
+          For every node object, the version number is incremented each time it is updated. It is required
```

Comment:
Wondering; this currently says `type: "object"`. Can we define a "type" for this so that we can have a central definition of `Version` and a single place to put the description?

/cc @bfirsh @dnephin